### PR TITLE
Fix likelihood documentation

### LINKED ISSUE #1345: [Documentation] No WhiteNoiseKernel?
In the [Exact GP Regression tutorial](https://docs.gpytorch.ai/en/v1.2.1/examples/01_Exact_GPs/Simple_GP_Regression.html), there's mention of a WhiteNoiseKernel, yet no such kernel currently exists. There was issue #113 and PR #152 regarding this, but it looks like it got removed at some point. Is there some alternative I should be using instead to add noise to my kernel?

### PR DESCRIPTION
**Pull request by gpleiss**
- [x] Add missing likelihoods to the docs
- [x] Remove references to WhiteNoiseKernel

[Closes #1345]

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -76,6 +76,8 @@ def __getattr__(cls, name):
             return _SubDistribution
         elif "Kernel" in name or "Parallel" in name:
             return _Kernel
+        elif "__version__" in name:
+            return "1000000000.0.1"
         else:
```
Comment (jacobrgardner) [inline docs/source/conf.py:81]: What?

File: examples/01_Exact_GPs/Simple_GP_Regression.ipynb [hunk 2]
```diff
@@ -96,14 +96,21 @@
     "This specification leaves a large amount of flexibility when defining a model. For example, to compose two kernels via addition, you can either add the kernel modules directly:\n",
     "\n",
     "```python\n",
-    "self.covar_module = ScaleKernel(RBFKernel() + WhiteNoiseKernel())\n",
+    "self.covar_module = ScaleKernel(RBFKernel() + LinearKernel())\n",
```
Comment (jacobrgardner) [inline examples/01_Exact_GPs/Simple_GP_Regression.ipynb:99]: Does this still produce a reasonable plot with a linear kernel?

**Review by jacobrgardner — APPROVED** [review]

File: examples/01_Exact_GPs/Simple_GP_Regression.ipynb (hunk 2 above)
Comment (gpleiss) [reply examples/01_Exact_GPs/Simple_GP_Regression.ipynb:99]: Yeah it still looks fine

File: docs/source/conf.py (hunk 1 above)
Comment (gpleiss) [reply docs/source/conf.py:81]: The doc build was failing because the mocked-out pytorch didn't have a `__version__` property. This fixes that :)

**Pull request merged**